live-examples/css-examples/cursor/cursor.css

```diff
@@ -0,0 +1,15 @@
+.example-choice-list {
+  height: 500px;
+  width: 250px;
+}
```

Comment:
I don't think we should be adjusting the editor styles in this CSS (but actually, you don't need this anyway).